src/rf/js/src/home/views.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+var React = require('react'),
+    asset = require('../core/utils').asset;
+
+var LoginView = React.createBackboneClass({
+    render: function() {
+        return <div>
+            <div className="block login-block">
+                <img src={asset('img/logo-raster-foundry.png')} />
+                <div className="form-group">
+                    <label htmlFor="search">Username</label>
+                    <input type="text" className="form-control" id="search" />
+                </div>
+                <div className="form-group">
+                    <label htmlFor="login-password">Password</label>
+                    <input type="password" className="form-control" id="login-password" />
+                </div>
+                <div className="form-action">
+                    <a href="#" className="btn btn-secondary btn-block"
+                        onClick={this.props.handleLogin}>Log In</a>
```

Comment:
Good point, I will use a form tag and the `onSubmit` event instead.

I know buttons and links aren't interchangeable but I know in the past styling `input`/`button` has been more difficult compared to styling anchors. I don't have a problem with changing this.